From acd955622826f7a2ea8cea40500e24552ca745d2 Mon Sep 17 00:00:00 2001
From: Guenter Roeck <linux@roeck-us.net>
Date: Sun, 21 Apr 2013 09:08:11 -0700
Subject: [PATCH 17/19] hwmon: (nct6775) Constify strings

nct6775_sio_names should be a constant pointer to an array of
constant strings.

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
(cherry picked from commit 6d4b3621bb613da51ae7474c50f5cb37465b6f37)
---
 drivers/hwmon/nct6775.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/hwmon/nct6775.c
+++ b/drivers/hwmon/nct6775.c
@@ -4035,7 +4035,7 @@ static struct platform_driver nct6775_dr
 	.remove		= nct6775_remove,
 };
 
-static const char *nct6775_sio_names[] __initconst = {
+static const char * const nct6775_sio_names[] __initconst = {
 	"NCT6775F",
 	"NCT6776D/F",
 	"NCT6779D",
